tests/e2e-textile.json

```diff
@@ -176,7 +176,7 @@
       "tre": 0.13206487622856444,
       "wtu": 2.0035853668394106,
       "ecs": 859.7119944122998,
-      "pef": 875.7274702532203
+      "pef": 875.7274702532204
```

Comment:
No big deal but do you have any clue why these have changed? Looks like floating point issues but can't track it to a specific reason.